tests/sentry/grouping/snapshots/test_variants/test_event_hash_variant/newstyle@2023_01_11/group_125_event_126.pysnap

```diff
@@ -119,7 +119,7 @@ app:
           frame (non app frame)
             function*
               "_CFBundleDlfcnLoadBundle"
-          frame (marked as prefix frame by stack trace rule (category:load +sentinel +prefix))
+          frame (marked out of app by stack trace rule (family:native package:/usr/lib/** -app))
```

Comment:
Since there's no hash change I can assume that there will not be a new group.

It seems the rules were clobbering the work done by `family:native package:/usr/lib/** -app`:
https://github.com/getsentry/sentry/blob/2238df57a679da00f46c4b9dbea6d13c951cb9d8/src/sentry/grouping/enhancer/enhancement-configs/newstyle%402023-01-11.txt#L13

For context, this is what I'm removing in this PR:
```
family:native function:dlopen category=load
category:load +sentinel +prefix
```